src/opera_disp_tms/generate_frame_tile.py

```diff
@@ -267,9 +287,11 @@ def main():
     """
     parser = argparse.ArgumentParser(description='Create a frame metadata tile for a given bounding box')
     parser.add_argument('bbox', type=int, nargs=4, help='Bounding box in the form of min_lon min_lat max_lon max_lat')
-    parser.add_argument('--ascending', action='store_true', help='Use ascending orbit pass')
+    parser.add_argument(
+        '--direction', type=str, choices=['ascending', 'descending'], help='Direction of the orbit pass'
```

Comment:
If this is going to be an optional `--` parameter, a default should be specified. Otherwise it should be a required positional parameter.

Reply:
Done!

Reply:
done